### PR TITLE
KaterinaTerekhovaCreateDescriptionOfFolderTest

### PR DESCRIPTION
**Pull request by KaterinaTerekhova**
Created test of TC_001_004

### CODE REVIEW

**Comment by KaterinaTerekhova** [comment]
[AT_011.0040 | New folder> add description](https://trello.com/c/FIWyPUdS/345-at0110040-new-folder-add-description)

File: src/test/java/CreateDescriptionOfFolderTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        getDriver().findElement(By.xpath("//div[@id='tasks']/div/span/a/span[2]")).click();
+        getDriver().findElement(By.name("name")).sendKeys("Financial Project");
+        getDriver().findElement(By.xpath("//div[@id='j-add-item-type-nested-projects']//li[1]")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.name("_.description"))
+                .sendKeys("The Company created new financial projec witn return investments");
+        getDriver().findElement(By.id("yui-gen6-button")).click();
+
+        Assert.assertEquals(descriptionText, getDriver().findElement(By.id("view-message")).getText());
+
```
Comment (Rom-T) [inline src/test/java/CreateDescriptionOfFolderTest.java:36]: Line 36 - better to delete

File: src/test/java/CreateDescriptionOfFolderTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * log in Jenkins
+ * - click "New Item"
+ * - create new folder  "Financial Project"
+ * - select "Folder"
+ * - click "Ok"
+ * - add description "The Company created new financial projec witn return investments"
+ * - click "Save"
+ * - make sure the description is displayed
+ */
+
```
Comment (Rom-T) [inline src/test/java/CreateDescriptionOfFolderTest.java:17]: it is better to delete the test case, since the PR contains a link to it in Trello

**Review by Rom-T — APPROVED** [review]

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/CreateDescriptionOfFolderTest.java (hunk 1 above)
Comment (KaterinaTerekhova) [reply src/test/java/CreateDescriptionOfFolderTest.java:36]: Thank you for your comments and your time

**Pull request merged**